Library/Homebrew/utils/svn.rb

```diff
@@ -31,8 +31,11 @@ def remote_exists?(url)
         return true unless available?
 
         # OK to unconditionally trust here because we're just checking if a URL exists.
-        system_command("svn", args: ["ls", url, "--depth", "empty",
-                                     "--non-interactive", "--trust-server-cert"], print_stderr: false).success?
+        args = ["ls", url, "--depth", "empty", "--non-interactive", "--trust-server-cert"]
+        if Version.create(version) >= Version.create("1.9")
+          args << "--trust-server-cert-failures=expired,not-yet-valid"
+        end
```

Comment:
```suggestion
        if Version.create(version) >= Version.create("1.9")
          args << "--trust-server-cert-failures=expired,not-yet-valid"
        end
```

This should also print a warning and I'd say ideally `Library/Homebrew/download_strategy.rb` should share this logic perhaps by making it a Utils::Svn helper method called by both?